scalableminds/webknossos
# Load histogram data sequentially

This PR adjusts the parallel sending of histogram data requests to a sequential procedure. That results in less blockage of bucket data requests.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open the dev tools of your browser
- open a tracing in webknossos
- the histogram requests should be sent one after the other
- the data requests should be sent parallel 

### Issues:
- fixes #4829

------
